public/app/features/alerting/unified/components/notification-policies/ContactPointSelector.tsx

```diff
@@ -12,30 +11,22 @@ import { ContactPointWithMetadata } from '../contact-points/utils';
 
 const MAX_CONTACT_POINTS_RENDERED = 500;
 
-// Mock sleep method, as fetching receivers is very fast and may seem like it hasn't occurred
-const sleep = (ms: number) => new Promise((resolve) => setTimeout(resolve, ms));
-const LOADING_SPINNER_DURATION = 1000;
-
 type ContactPointSelectorProps = {
   selectProps: SelectCommonProps<ContactPointWithMetadata>;
-  showRefreshButton?: boolean;
   /** Name of a contact point to optionally find and set as the preset value on the dropdown */
   selectedContactPointName?: string | null;
   onError?: (error: Error) => void;
 };
 
 export const ContactPointSelector = ({
   selectProps,
-  showRefreshButton,
   selectedContactPointName,
   onError = () => {},
 }: ContactPointSelectorProps) => {
   const { selectedAlertmanager } = useAlertmanager();
```

Comment:
Worth throwing an error or something here if it's being used for `grafana` alertmanager?

Reply:
I'm on the fence for this one because it _might_ introduce runtime errors because we accidentally imported this component somewhere in a Grafana AM context.